### DISCUSSION TITLE
LaTeX Warning: Hyper reference

### DISCUSSION THREAD

**Discussion by PsychoPunkSage** [Cyfrin/security-and-auditing-full-course-s23#39] [Q&A]
Hi,
I have downloaded LATEX and Pandoc successfully. But I'm getting error on converting "report.md" to "report.pdf"
Command:: 
```bash 
pandoc report.md -o report.pdf --from markdown --template=eisvogel --listings
```

Error::
```
[WARNING] [makePDF] LaTeX Warning: Hyper reference
  `h-1-anything-stored-on-chain-can-be-accessed-by-anyone-solidity-access-control-ie-private-public-etc-is-only-applicable-for-contracts'
  on page 2 undefined on input line 458.
[WARNING] [makePDF] LaTeX Warning: There were undefined references.
```
Though the iwass able to get pdf but can anyone tell me what is the issue here and how to solve it....
**Report Repo Link:: https://github.com/PsychoPunkSage/Security_audit/tree/Audits/Checksums/1-passwordstore-audit/audit-data**

Thank You

**Comment by PatrickAlphaC** [comment]
The error message you're seeing from Pandoc and LaTeX indicates that there's an issue with hyperreferences within your Markdown document. Specifically, it seems to be warning about an undefined reference, which typically happens if you have a hyperlink or internal link that points to a target that doesn't exist or isn't properly defined.

The reference in the warning message suggests it's looking for a header with the ID h-1-anything-stored-on-chain-can-be-accessed-by-anyone-solidity-access-control-ie-private-public-etc-is-only-applicable-for-contracts, which might be incorrectly formatted or missing in your document.

_Response created with help from ChatGPT_

**Comment by rube-de** [comment]
I had the same warning and no linking in the TOC of the pdf 
I had a . at the end of the title, after removing the "." from the title the warning goes away and the pdf is created correctly with working hyperlinks

**Reply by cromewar** [reply]
You are nailing it @rube-de, amazing!
